unreal/src/lib.rs

```diff
@@ -194,6 +214,29 @@ impl Unreal4Crash {
     pub fn get_context(&self) -> Result<Option<Unreal4Context>, Unreal4Error> {
         Unreal4Context::from_crash(self)
     }
+
+    /// Get the log messages of this crash.
+    pub fn get_logs(&self) -> Result<Vec<Unreal4LogEntry>, Unreal4Error> {
+        Ok(match self.get_file_slice(Unreal4FileType::Log)? {
+            Some(f) => {
+                let reader = BufReader::new(f);
+                let mut logs = Vec::new();
+                for line in reader.lines() {
+                    let line = line.map_err(Unreal4Error::InvalidLogEntry)?;
+                    let entry = LogEntry::parse(line.as_bytes());
+                    let (component, message) = entry.component_and_message();
+                    logs.push(Unreal4LogEntry {
+                        timestamp: entry.utc_timestamp(),
+                        component: component,
+                        message: message,
+                    });
+                }
+                logs.reverse();
```

Comment:
Isn’t the reverse wrong here?

Reply:
Latest log entry shows up at idx 0 (the unit test shows that) as if it was a queue of messages.
Rather drop the `reverse`?

Reply:
removed